source/agora/crypto/Schnorr.d

```diff
@@ -230,6 +230,25 @@ public Signature sign (T) (
      */
     // Compute the challenge and reduce the hash to a scalar
     Scalar c = hashFull(const(Message!T)(X, R, data));
+    return sign(x, R, r, c);
+}
+
+/// sign with predefined message hash c (used for multisig with threshold)
+public Signature sign (in Scalar x, in Point R, in Scalar r, in Scalar c)
```

Comment:
I think the last parameter `c` should have a logical name. Its meaning is confused with other parameters, `x` or `r`.

Reply:
As c is defined in the comment above the declaration I would prefer to keep it as c. Otherwise the contents of the function get difficult to read. I changed the comment to use `c` rather than c to help make it clearer. Also in Schnorr discussions the hashed message is often referred to as `c`.